language/move-prover/tests/sources/functional/unused_schema.exp

```diff
@@ -0,0 +1,25 @@
+Move prover returns: exiting with boogie verification errors
+warning: A schema is not used: TestUnusedSchema::AddsThree
+
+    ┌── tests/sources/functional/unused_schema.move:21:5 ───
+    │
+ 21 │ ╭     spec schema AddsThree {
+ 22 │ │         i: num;
+ 23 │ │         result: num;
+ 24 │ │         ensures result == i + 3;
+ 25 │ │     }
+    │ ╰─────^
+    │
+
+error:  A postcondition might not hold on this return path.
```

Comment:
Ah! It is actually exiting with verification errors as well. I suggest we just remove those errors to keep the tests clean. We may(?) need to fix the test driver in `tests/testsuite.rs` a bit to handle the warnings case correctly.

Reply:
I think we can leave as is right now and change the driver when more use cases for warnings come up.

Reply:
That's the kind of thing which leads to entropy in code. "Later" will then easily become "never". I also don't know whether you need to do anything at the driver at all, my main issue here is that you have a verification error in tests around warnings and that does not make sense -- or did you do it for some purpose?

Reply:
Also, if the prover prints "exiting with verification errors" when you actually have only warnings is kind of a bad user experience. So this is not just about the tests.

Reply:
Well if there are only warnings the prover will not print out "exiting with verification errors". It will only print out the warnings. The case that might lead to confusion is when there are both warnings and errors. Then prover would say "exiting with errors", followed by warning messages and error messages.

Reply:
I have updated the test case so there is only a warning. And the exp file only contains the warning message.